src/components/toolbar/toolbar.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Component,
+  ChangeDetectionStrategy
+} from 'angular2/core';
+import {Renderer} from 'angular2/core';
+import {ElementRef} from 'angular2/core';
+
+@Component({
+  selector: 'md-toolbar',
+  inputs: ['color'],
+  templateUrl: './components/toolbar/toolbar.html',
+  styleUrls: ['./components/toolbar/toolbar.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MdToolbar {
+
+  private _color: string;
+
+  constructor(private elementRef: ElementRef, private renderer: Renderer) {}
+
+  set color(value: string) {
+    if (this._color) {
```

Comment:
Using the truithiness of values doesn't work in Dart, so you have to explicitly check for `== null` and `== ''`